ntbs-service/Pages/ClinicalTimelines/Edit.cshtml

```diff
@@ -3,7 +3,9 @@
 @using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
 
 @{
+    Layout = "../Shared/_NotificationLayout.cshtml";
     ViewData["Title"] = "Notification - Clinical Timeline";
+    ViewData["PageType"] = "clinicalDetails";

```

Comment:
This is used for string matching to see which page is "active" on the contents list on the left, PageType and Title are similar, unsure if I should merge into one and string match the longer Title field. Another possibility is to use separate ViewData["clinicalDetails"] = true; for each page instead of one changing variable.

Reply:
Sure I'll implement that, I'll see when Farman's work gets on master and rebase off of his if he can get it ready before me